src/lib/layout/build.ts

```diff
@@ -105,10 +105,12 @@ function getBuildOutputDir(
   buildOutput: string | undefined,
   scanResult: ScanResult
 ): string {
-  const output =
-    buildOutput ??
-    scanResult.tsConfig.content.options.outDir ??
-    DEFAULT_BUILD_DIR_PATH_RELATIVE_TO_PROJECT_ROOT
+  // todo normalize because ts in windows is like "C:/.../.../" instead of "C:\...\..." ... why???
+  const outDir = scanResult.tsConfig.content.options.outDir
```

Comment:
Had to deal with this several times throughout. For some reason I cannot figure out so far the tsc API when processing the tsconfig creates [seemingly] invalid windows paths. This leads to test problems where I rely on matching using `ctx.fs.path` or `ctx.fs.cwd()`. I assume it must work right now so haven't attempted to "fix" this throughout the codebase, instead just a few places where it breaks the tests. The TODO comments are here for us to revisit when we have more time.

I might make an issue on the TS repo about this.

Reply:
nevermind... https://github.com/graphql-nexus/nexus/pull/1123